src/main/java/ratpack/zipkin/RequestAnnotationExtractor.java

```diff
@@ -16,14 +16,18 @@
 package ratpack.zipkin;
 
 import com.github.kristofa.brave.KeyValueAnnotation;
+import java.util.Arrays;
 import ratpack.http.Request;
 
 import java.util.Collection;
 import java.util.Collections;
+import zipkin.TraceKeys;
 
 @FunctionalInterface
 public interface RequestAnnotationExtractor {
   Collection<KeyValueAnnotation> annotationsForRequest(Request request);
 
-  RequestAnnotationExtractor DEFAULT = (request -> Collections.emptyList());
+  RequestAnnotationExtractor DEFAULT = (request -> Collections.singletonList(
+      KeyValueAnnotation.create(TraceKeys.HTTP_URL, request.getPath())));
```

Comment:
I'm going to switch this to HTTP_PATH but feel free to request otherwise if you feel there is a good reason.

Based on experience I've found searching by path more meaningful and that including the host can make searching harder.  The reason for this is that most Zipkin storage components allow searching by value prefix.  If your host varies for any reason it makes searching by prefix a lot harder.

Having the query parameters in the key could be interesting though a bit scary from both a span storage perspective as well as a security perspective.  Hopefully most folks don't have giant query strings or sensitive data in their query strings but they do have a way of creeping in on occasion. 

Again just my 2 cents so please let me know if anyone feels otherwise.